### ISSUE TITLE
Previews for the documents

### ISSUE THREAD

**Issue by jancborchardt** [owncloud/richdocuments#94]
Is it planned/possible to have previews of the documents shown in the app? Cause right now it’s just a grid of giant icons – would be cool to make that a bit nicer. :)

The idea would be to still have the grid, but a big preview and then the filetype icon a bit smaller in the corner.

cc @timar @pranavk and @LukasReschke for security

**Comment by pranavk** [comment]
We have a lool/convert-to API[1] which can give you converted documents in any other format. So if you do a format=png there, we get the png of the first page of the document. I think that can be used here as previews ?

Please note that there was a regression in that API, so it wasn't converting the documents properly, but it has been fixed recently, so it works in current master, collabora-online-1-0 branches now.

[1] https://gerrit.libreoffice.org/gitweb?p=online.git;a=blob;f=loolwsd/reference.txt;h=bc0cbe8559a01789747adfdc2fdbcb47af3fb3b4;hb=HEAD

**Comment by jancborchardt** [comment]
Yeah, that would be quite awesome! PNG of the first page is exactly what we need.
